server/src/main/java/org/opensearch/indices/replication/common/CopyState.java

```diff
@@ -83,7 +67,12 @@ public IndexShard getShard() {
         return shard;
     }
 
-    public ReplicationCheckpoint getRequestedReplicationCheckpoint() {
```

Comment:
This field is no longer required, this was used as the cache key in OngoingSegmentReplications.